app/script/main/app.coffee

```diff
@@ -239,7 +239,7 @@ class z.main.App
       if is_reload and error.type not in [z.client.ClientError.TYPE.MISSING_ON_BACKEND, z.client.ClientError.TYPE.NO_LOCAL_CLIENT]
         @auth.client.execute_on_connectivity(z.service.Client::CONNECTIVITY_CHECK_TRIGGER.APP_INIT_RELOAD).then -> window.location.reload false
       else if navigator.onLine
-        @logger.error "Caused by: #{error?.message or error}"
+        @logger.error "Caused by: #{error?.message or error}", error
```

Comment:
Problematic. If error is a string as it sometimes is and not the error object, we will end up with a log message that does no longer contain the "Caused by reference". Maybe we should be more explicit and do a proper _.isObject check with tow error logging cases.